src/containers/diseasePage/index.js

```diff
@@ -70,7 +70,10 @@ class DiseasePage extends Component {
 
     const title = disease.name || disease.id;
 
-    let keywords = ['disease', data.id, data.name, ...data.synonyms, data.definition];
+    let keywords = ['disease', data.id, data.name, data.definition];
+    if(data.synonyms){
+      keywords.push(data.synonyms);
```

Comment:
This is going to put the put the synonym array itself into the array of keywords:
`['disease', 'DO:12345', 'foo', ['synonym 1', 'synonym 2']]`

Did you mean to use `.concat()` instead of `.push()`?

Reply:
should be:

    keywords.push(...data.synonyms)

Fixed now.

Reply:
That works too!